ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -1658,3 +2135,65 @@ public void close() {
     return futureResult;
   }
 }
+
+/**
+ * A singleton implementation of {@link AccountServiceFactory} that returns an instance of {@link FrontendTestAccountService}.
+ */
+class FrontendTestAccountServiceFactory implements AccountServiceFactory {
+  private AccountService accountService = null;
+
+  @Override
+  public AccountService getAccountService() throws InstantiationException {
+    if (accountService == null) {
```

Comment:
Is the factory required here? We pass the `AccountService` directly to the `BlobStorageService`, right?